utils/reader.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+
```

Comment:
propose add dict with callables like: readers = {'.json': read_json, '.txt': read_txt} and method which can choose needed reader from dict